src/renderers/WebGLCubeRenderTarget.js

```diff
@@ -30,9 +31,6 @@ class WebGLCubeRenderTarget extends WebGLRenderTarget {
 
 		this.texture.isRenderTargetTexture = true;
 
-		this.texture.generateMipmaps = options.generateMipmaps !== undefined ? options.generateMipmaps : false;
-		this.texture.minFilter = options.minFilter !== undefined ? options.minFilter : LinearFilter;
-
```

Comment:
These are default values inherited from `RenderTarget`. Should we use `NearestFilter` as the default for `WebGL3DRenderTarget` and `WebGLArrayRenderTarget`? https://github.com/mrdoob/three.js/issues/28100#issuecomment-2049256804

Reply:
If you create an instance of `CubeTexture`, `generateMipmaps` is `true` and  `minFilter` is `LinearMipmapLinearFilter`. Without the above lines, these values are not changed to honor the the usage in context of a cube render target. So `generateMipmaps` should be `false` and `minFilter` set to `LinearFilter`. This should work:
```js
this.texture.setValues( {
  minFilter: LinearFilter,
  generateMipmaps: false,
  ...options
} );
```
> Should we use NearestFilter as the default for WebGL3DRenderTarget and WebGLArrayRenderTarget?

Yes, but looking at the change this is already the case, right? When `WebGLArrayRenderTarget` creates and instance of `DataArrayTexture`, the filter properties already hold `NearestFilter` as the value.

Reply:
Yes, that seems right.

FYI: I've deleted my previous comment since I had to read the change to better understand it^^.

Reply:
I think I'm still wrong here, since `RenderTarget` creates its own texture whose configuration I was referring to here. We can make `texture` an option and pass the instantiated `CubeTexture` etc. if we really wanted to inherit the configuration there.